doc/tutorials/README.txt

```diff
@@ -0,0 +1 @@
+FlamingPy Tutorials

```

Comment:
Should we just remove this line (since it's already obvious that these are tutorials), or would you rather expand this with some more details? @ilan-tz

Reply:
I agree! Let's get rid of it

Reply:
Done!